profile/readme.md

```diff
@@ -0,0 +1,19 @@
+# ROS-Industrial
+
+Welcome to the ROS-Industrial GitHub organization.
+
+## Purpose
+
+The ROS-Industrial open source project is seeks to provide modules and resource to enable industrial relevant hardware, tools and application leveraging the ROS ecosystem. The purpose of this organization is to track foundational capabilities and resources that are community leveraged, maintained and fully open source. This GitHub organization supports both community and consortia generated projects, modules and resources to enable industrial leverage of ROS and ROS 2 specifically, in industrial settings.
```

Comment:
```suggestion
The ROS-Industrial open source project seeks to provide modules and resources to enable industrial relevant hardware, tools and applications leveraging the ROS ecosystem. The purpose of this organization is to track foundational capabilities and resources that are community leveraged, maintained and fully open source. This GitHub organization supports both community and consortia generated projects, modules and resources to enable industrial leverage of ROS and ROS 2 specifically, in industrial settings.
```

Reply:
This sentence also seems to duplicate *industrial*:

> This GitHub organization supports both community and consortia generated projects, modules and resources to enable industrial leverage of ROS and ROS 2 specifically, in industrial settings.

Reply:
```suggestion
The ROS-Industrial open source project is seeks to provide modules and resource to enable industrial relevant hardware, tools and application leveraging the ROS ecosystem. The purpose of this organization is to track foundational capabilities and resources that are community leveraged, maintained and fully open source. This GitHub organization supports both community and consortia generated projects, modules and resources to enable the use of ROS and ROS 2, specifically in industrial settings.
```